src/it/java/org/eclipse/microprofile/starter/utils/Whitelist.java

```diff
@@ -36,8 +36,11 @@
             Pattern.compile(".*com.hazelcast.nio.tcp.TcpIpConnectionErrorHandler.*"),
     }),
     LIBERTY("liberty", new Pattern[]{
-        Pattern.compile(".*FrameworkEvent ERROR.*"),
-        Pattern.compile(".*CWWKE0701E.*"),
+            Pattern.compile(".*FrameworkEvent ERROR.*"),
+            Pattern.compile(".*CWWKE0701E.*"),
+            // An exception occurred while stopping the application liberty. The exception message was: java.lang.NoClassDefFoundError: com/ibm/ws/threading/internal/ImmediateFutureImpl
+            Pattern.compile(".*CWWKZ0010E:.*"),
+            Pattern.compile(".*Could not load service class com.ibm.ws.io.smallrye.graphql.component.GraphQLExtension.*")
```

Comment:
@andymc12 when will the warning be fixed?

Reply:
Good question - I hadn't noticed this before.  I'll open an issue and look into why the warning is occurring.  Thanks for mentioning it.